vaas-app/src/vaas/cluster/cluster.py

```diff
@@ -59,7 +59,7 @@ def load_vcl(self, start_processing_time, clusters):
                 cluster.reload_timestamp = start_processing_time
                 cluster.save()
         except VclLoadException as e:
-            self.logger.error('Loading error: {} - rendered vcl-s not used'.format(e))
+            parallel_loader.discard_loaded_unused_vcl(vcl_list)
```

Comment:
In this way we loose information about connectivity problems with varnish (look at line #78)